backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/StartClusterUpgradeCommand.java

```diff
@@ -46,7 +46,12 @@ protected StartClusterUpgradeCommand(Guid commandId) {
     @Override
     protected void executeCommand() {
         TransactionSupport.executeInNewTransaction(() -> {
-            boolean updated = clusterDao.setUpgradeRunning(cluster.getId());
+            String correlationId =
+                StringUtils.isNotBlank(getParameters().getUpgradeCorrelationId())
+                    ? getParameters().getUpgradeCorrelationId()
+                    : getParameters().getCorrelationId();
+
+            boolean updated = clusterDao.setUpgradeRunning(cluster.getId(), correlationId);
             if (updated) {
                 log.info("Successfully set upgrade running flag on cluster '{}'.", cluster.getId());
```

Comment:
Is this message really needs to be at INFO level? It seems more to me like a DEBUG stuff

Reply:
Ack